src/safeds/ml/classical/classification/_ada_boost.py

```diff
@@ -13,27 +13,34 @@
 
 
 class AdaBoost(Classifier):
-    """Ada Boost classification.
+    """
+    Ada Boost classification.
 
     Parameters
     ----------
     learning_rate : float
         Weight applied to each classifier at each boosting iteration. A higher learning rate increases the contribution
         of each classifier. Has to be greater than 0.
+    maximum_number_of_learners: int
```

Comment:
For the same reason as [here](https://github.com/Safe-DS/Stdlib/pull/268/files#r1185937745), the parameters should be swapped. Same in other places `learning_rate` and `maximum_number_of_learners` occur.